README.md

```diff
@@ -475,7 +475,7 @@ To do so manually:
 Example:
 
 ```Shell
-MAINTENANCE_MESSAGE='{"active": true, "message": "SimpleReport is currently experiencing service degradation"}' MAINTENANCE_ENV=dev yarn run maintenance:start
+MAINTENANCE_MESSAGE='{"active": true, "header" : "SimpleReport is currently experiencing an outage.", "message": "SimpleReport is currently experiencing service degradation"}' MAINTENANCE_ENV=dev yarn run maintenance:start
```

Comment:
This reads weirdly to me - the header paired with the message more or less repeat each other. Also, the header and message should swap the period. @Rebecca-LM any default copy ideas? I'd think the header should set out the general case and the message should clarify with what details we have/can provide. Maybe the default message is just blank?

Reply:
oh that's just an example, we changed the default header to "**SimpleReport alert:**"

Reply:
Btw, this is how it was before, The header was always hardcoded to 
**SimpleReport is currently experiencing an outage.**
This PR will change that to make it configurable from the `maintenance.json`
and if header is not specified, it will default to **SimpleReport alert:**